Cargo.lock

```diff
@@ -4856,16 +4862,17 @@ checksum = "e3d7ddaed09e0eb771a79ab0fd64609ba0afb0a8366421957936ad14cbd13630"
 
 [[package]]
 name = "polling"
-version = "3.4.0"
+version = "3.7.4"
```

Comment:
so u did this manually or what? :thinking:

Reply:
No, `cargo update -p polling`. The cargo dependency resolver does it's best to choose the right versions but it's not always right.

Reply:
afaik, If we depend on crates A and B, but crate A depends on 0.x version of another crate while crate B depends on 0.y, then the only thing cargo can do is compile both. If we updated crate A to a version that uses 0.y then cargo removes 0.x version.

Reply:
![image](https://github.com/user-attachments/assets/ef64c8c6-89b8-4312-8cfb-14c148163843)
this is the net change btw. `hermit-abi` is not duplicated but `windows-sys` isn't.
`windows-sys` is larger and we get more deps updated with this change, so good.

Reply:
Additionally, `hermit-abi` is duplicated due to 2 somewhat independent dependencies (or repos) as updating libp2p leads to this duplication indirectly but before the commit `windows-sys` was duplicated due to updating libp2p only.

Reply:
so we removed one duplication to get another one? :thinking:

fwiw current one is 200kb bigger on size.

Reply:
> fwiw current one is 200kb bigger on size.

that's the download count i think

Reply:
> that's the download count i think

It is

Reply:
https://crates.io/crates/windows-sys is `2.28 MiB`

Reply:
@onur-ozkan please resolve this comment and this https://github.com/KomodoPlatform/komodo-defi-framework/pull/2326#discussion_r1925146861 when you think it's fine so that I can approve and merge the PR